### PR TITLE
download the bahmni installer & install

### PR DESCRIPTION
**Pull request by munnavhai**

### CODE REVIEW

File: artifacts/.gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 bahmni-playbooks/
 *_config/
 misc*
+misc/.gitkeep
```
Comment (R-VdP) [inline artifacts/.gitignore:5]: This doesn't make much sense, we use these .gitkeep files to be able to include empty directories.
Why did you add this line?

**Review by R-VdP — CHANGES_REQUESTED** [review]

File: artifacts/.gitignore (hunk 1 above)
Comment (munnavhai) [reply artifacts/.gitignore:5]: misc is a soft link to the most recent misc directory (currently its misc-4). If we create any file then it becomes a directory instead of a softlink which causes issues while building new docker image because we defined copying the files in Dockerfile from misc and we do not want to change Dockerfile every time when we need different misc directory.

Comment (R-VdP) [reply artifacts/.gitignore:5]: Ok, but in that case you can just remove the .gitkeep file, which will also remove the directory. There is no need to add it to the .gitignore file.

Comment (munnavhai) [reply artifacts/.gitignore]: Done

**Review by R-VdP — APPROVED** [review]
